Добавить импорт из базы данных

mikhailmineev opened the pull request:
Связанные с пулреквестом задачи:
- [Добавить импорт из базы данных #578](https://github.com/aurusov/raox/issues/578)
- [Добавить эксперименты #592](https://github.com/aurusov/raox/issues/592)
- [ConsoleView сильно зависает при логировании большого количества строк #594](https://github.com/aurusov/raox/issues/594)

Commit:
Библиотека querydsl теперь подтягивается в проекты (aurusov/raox#578)

Commit:
В моделях появилась возможность подключаться к базе MySQL (aurusov/raox#578)

Commit:
Данные успешно читаются из базы (aurusov/raox#578)

Commit:
Добавил dataprovider и dataentity в РДО (aurusov/raox#578)

Commit:
Поправил SqlDataProvider (aurusov/raox#578)

Commit:
Добавил в язык РДО dataentity (aurusov/raox#578)

Commit:
Модели описываются в java/domain (aurusov/raox#578)

Commit:
Добавил генерацию Query классов из QueryDSL (aurusov/raox#578)

Commit:
Поправил форматирование (aurusov/raox#578)

Commit:
Перенес querydsl в отдельный плагин (aurusov/raox#578)

Commit:
Переделал частично генерацию QueryDSL (aurusov/raox#578)

Commit:
Исправил querydsl плохо работал при отсутствии уже сгенереных классов (aurusov/raox#578)

Commit:
Revert "Исправил querydsl плохо работал при отсутствии уже сгенереных классов (aurusov/raox#578)"

This reverts commit 60314216dd9140017ff0101c2832f87f41bfd550.

Commit:
Fix after revert (aurusov/raox#578)

Commit:
Исправления по замечаниям (aurusov/raox#578)

Commit:
Добавил типы sursor и variable (aurusov/raox#578)

Commit:
Рефакторинг кода, генерирующего querydsl (aurusov/raox#578)

Commit:
Выпилил cursor, оставил variable (aurusov/raox#578)

Commit:
Включил валидатор при сохранении, поскольку иначе модели с Query
классами не запускаются (aurusov/raox#578)

Commit:
Добавил сброс времени симуляции (aurusov/raox#578)

Commit:
Revert "Добавил сброс времени симуляции (aurusov/raox#578)"

This reverts commit 7f11627374952f74ccfa4291aad49cb55955ab0d.

Commit:
Revert "Выпилил cursor, оставил variable (aurusov/raox#578)"

This reverts commit f905189627ff9bddac1c3918973fc54b5be2b3c6.

Commit:
Revert "Рефакторинг кода, генерирующего querydsl (aurusov/raox#578)"

This reverts commit e3afb209fab6a7462b48c8cd4e50155c9f728399.

Commit:
Fix after failed revert (aurusov/raox#578)

Commit:
Отрефакторил код для поддержки простых экспериментов (aurusov/raox#592)

Commit:
Исправлено ConsoleView сильно зависает при логировании большого
количества строк (aurusov/raox#594)

Commit:
Merge branch 'release/current' into feature/database

AlexChernov reviewed (commented):
Minor:
По готовности кода, сделать rebase на release/current причесать историю коммитов, убрать пулы из мастера в ветку и т д. Текущий пулл реквест можно будет закрыть без подливания в мастер

AlexChernov reviewed (changes requested):
Minor: Возможно я вне контекста, но  ad25c30d085d294edee6e740832feee8620bd122 выглядит как коммит, который надо сделать в отдельную ветку и ревьювить в отдельном пул-реквесте

Commit:
Maven build fix (aurusov/raox#578)

Commit:
Убрал коннектор mysql, добавив возможность загружать библиотеки внутри
проектов (aurusov/raox#578)

Commit:
Добавлена возможность указывать любые папки с исходниками java
(aurusov/raox#578)

Commit:
Добавлен метод finish в модель, вызывающийся после окончания симуляции
(aurusov/raox#578)

Commit:
Переписал логгер (aurusov/raox#594)

- каждые 500 мс проверка текста на обновление
- скроллинг в конец при обновлении текста

Commit:
Рефакторинг извлек задержку в константу (aurusov/raox#594)

Commit:
Добавил в документацию правила смены версий (aurusov/raox#578)

Commit:
Исправлено querydsl не добавляется при упаковке в архив
(aurusov/raox#578)

Commit:
Исправлена ошибка при сборке, мавен записывал querydsl скомпилированный
jar в исходный (aurusov/raox#578)

Commit:
При сборке ассамблеи .queydsl библиотека копируется от исходной (aurusov/raox#578)

Commit:
Merge branch 'feature/experiments' into feature/database

# Conflicts:
#	ru.bmstu.rk9.rao.ui/src/ru/bmstu/rk9/rao/ui/execution/ExecutionJobProvider.java
#	ru.bmstu.rk9.rao/src/ru/bmstu/rk9/rao/validation/DefaultMethodsHelper.java

Commit:
Поддержка множественных экспериментов (aurusov/raox#592)

Commit:
Очистка кэша hibernate (aurusov/raox#592)

Commit:
Ошибка, при завершении прогона остаются события, идущие в другой и вся
логика отчистки прогона в finish (lekaitow/raox#592)
